eng/pipelines/templates/jobs/ci.yml

```diff
@@ -229,11 +234,23 @@ jobs:
   - template: /eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml
     parameters:
       JobTemplatePath: /eng/pipelines/templates/jobs/ci.tests.yml
+      ProjectListOverrideFile: ${{ parameters.ProjectListOverrideFile }}
       MatrixConfigs: ${{ parameters.MatrixConfigs }}
       MatrixFilters: ${{ parameters.MatrixFilters }}
       MatrixReplace: ${{ parameters.MatrixReplace }}
       CloudConfig:
-        Cloud: Public
+        Cloud: public
+      ${{ if ne(parameters.TestTargetDependency , 'not-specific') }}:
+        SparseCheckoutPaths: 
+          - "/sdk"
+          - "/tools/bootstrapTools/bootstrap.targets"
+          - "!/sdk/*/**/SessionRecords"
+        PreGenerationSteps: 
+        - template: /eng/pipelines/templates/steps/split-tests.yml
+          parameters:
+            TestTargetDependency: ${{ parameters.TestTargetDependency }}
+            ProjectListOverrideFile: ${{ parameters.ProjectListOverrideFile }}
```

Comment:
Just pass the exact same hardcoded value here as you do in to the tests-generate.yml file.

Reply:
![image](https://user-images.githubusercontent.com/48036328/180868957-d3c1f7b0-c875-46f0-aa06-b90167de0e2c.png)
Yes, hard-coded the same value in publish and download step